sunrisemvmtnyc/legislation-tracker
# feat: Get all filters working together

Legislator filter still using dummy data, but you can test the others:

TO TEST:
- initial page load: only shows sunrise bills
- change BILL STATUS filter: filters to only show correct bills
  - should also work with multiple options selected in BILL STATUS 
- change campaign: shows bills within campaign
  - should work with multiple options selected 
- both filters active: shows correctly filtered bills
